docs/pages/reference/backends.mdx

```diff
@@ -946,7 +946,32 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
-## Athena
+## Athena (Preview)
```

Comment:
Are we still marking features as in preview? CC @r0mant

Reply:
For this specific one, it's OK, because we know it has a serious implementation limitation.